R/reexports.R

```diff
@@ -1,4 +1,33 @@
+#' @importFrom bayestestR equivalence_test
+#' @export
+bayestestR::equivalence_test
+
 #' @export
 #' @importFrom datawizard standardize
 datawizard::standardize
 
+
+
+
+
+## TODO: removed the following from see / modelbased --------------------
+
+#' @export
+#' @importFrom datawizard ranktransform
+datawizard::ranktransform
+
+#' @export
+#' @importFrom datawizard adjust
+datawizard::adjust
+
+#' @export
+#' @importFrom datawizard change_scale
+datawizard::change_scale
```

Comment:
Why are you re-exporting the alias `change_scale()`, which will be removed from `{datawizard}` in future releases?

https://github.com/easystats/datawizard/issues/53

Reply:
Because modelbased is importing it from effectsize 🙃 and I don't want to break it.

Reply:
Argh, all these re-exports are making our lives so difficult. We really need to take care of this roadblock soon.